### ISSUE TITLE
"no lighting" ships.tbl flag no longer works as designed

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fs2open.github.com#3999]
The "no lighting" flag was added in 2011 by Valathil for the purpose of drawing ship textures exactly as they appear in the original graphics image, without applying mission lighting or effects.  It was first released in 3.6.14, and it is intended for ships that are treated as "special effects".  It can most obviously be seen in the Just Another Day 2.2x trilogy where it is applied to billboards, power-ups, and the Nyan cat.

The flag no longer works as designed because all ships are now passed through the same pipeline, so they all have lighting applied to them whether they have the `MR_NO_LIGHTING` flag or not.  It appears that the flag was not accommodated during one of the recent rendering upgrades.

In addition to the JAD trilogy, I'm working on a mod that could really benefit from having this flag fixed.

**Comment by EatThePath** [comment]
I worked on this with @Goober5000 some last night and determined that the flag actually does work, at least in some cases. I haven't fully traced out how it works, or checked on all the JAD examples to see how they're affected, but I can see definite differences in the lighting of ships with and without these flags. I think there's still an element of specular reflection occurring though, and if that can be eliminated that would probably be good. 

That said pre-PBR/deffered rendering mods with highly deliberate visual styles all suffer from what FSU artists have been complaining about for years, the HDR lighting and post-processing using the Uncharted 2 tonemapper. It will tend to make images designed for the old pipeline look washed out or faded towards white, and bright areas may get hit with bloom if it's enabled. Using a lighting profile with linear tonemapping at 1.0 exposure would likely approximate their pre-HDR look, though think it risks unsightly artifacting if bright lights(say multiple explosions and weapon glows or a level with multiple bright suns) max out the color somewhere. That may be the lesser of evils in old mods though.

Playing the math a UC2 tonemapper with around 2.6 exposure might avoid the washed out look for dark areas when compared to linear, but I haven't tried this out yet to see.

**Comment by z64555** [comment]
According to EatThePath on discord, the last estimated time this worked right was around 3.7.2

**Comment by The-E** [comment]
TL;DR: This issue is invalid, there is no way to get the old look back.

In a deferred/PBR rendering environment, "no lighting" makes no sense and cannot be made to make sense without adding in a bunch of special case handling in the entire graphics pipeline. The way we approximate the behaviour now (by treating a material with the "no lighting" flag as a glowing material) is the best we're going to get.

The correct way to handle this sort of asset would be to use scripting to render them as part of the HUD rendering.

**Comment by MoerasGrizzly** [comment]
> That said pre-PBR/deffered rendering mods with highly deliberate visual styles all suffer from what FSU artists have been complaining about for years, the HDR lighting and post-processing using the Uncharted 2 tonemapper. It will tend to make images designed for the old pipeline look washed out or faded towards white, and bright areas may get hit with bloom if it's enabled. Using a lighting profile with linear tonemapping at 1.0 exposure would likely approximate their pre-HDR look, though think it risks unsightly artifacting if bright lights(say multiple explosions and weapon glows or a level with multiple bright suns) max out the color somewhere. That may be the lesser of evils in old mods though.

There's  a way around that: Use the PPC tonemapper, set exposure to 1, toe strength to zero and shoulder strength to 1.
